packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -264,7 +265,49 @@ const pushTransaction = () => async (dispatch: Dispatch, getState: GetState) =>
             dispatch(accountsActions.updateAccount(pendingAccount));
             if (account.networkType === 'cardano') {
                 // manually add fake pending tx as we don't have the data about mempool txs
-                dispatch(addFakePendingTxThunk({ precomposedTx, txid, account: pendingAccount }));
+                dispatch(
+                    addFakePendingTxThunk({
+                        precomposedTx,
+                        txid,
+                        account: pendingAccount,
+                        txType: 'sent',
+                    }),
+                );
+            }
+
+            // tmp: creating pending transaction with local data. at this moment, we do it only
+            // for bitcoin-join account
+            // todo: it could be useful for all accounts but it requires some more work and refactoring
+            if (account.networkType === 'bitcoin') {
+                const { accounts } = getState().wallet;
+                // @ts-expect-error
+                const { outputs } = precomposedTx.transaction;
+                if (precomposedTx.transaction.outputs) {
+                    const targetAccounts = outputs.reduce<Account[]>((result, output) => {
+                        const targetAccount = accounts.find(
+                            account =>
+                                account.addresses?.used.find(
+                                    address => address.address === output.address,
+                                ) ||
+                                account.addresses?.unused.find(
+                                    address => address.address === output.address,
+                                ),
+                        );
+                        // only if we are 'feeding' own coinjoin account
+                        if (targetAccount?.accountType === 'coinjoin') {
```

Comment:
i think we can safely add it for all account types, response of `accountUpdate` will should remove this fake tx

Reply:
discussed, we are staying with 'coinjoin' for now. adding some issue as well